docs/features/dynamic_tasks.md

```diff
@@ -0,0 +1,100 @@
+# **Dynamic tasks**
+
+DAG Factory supports Airflow’s
+[Dynamic Task Mapping](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/dynamic-task-mapping.html),
+enabling workflows to dynamically create tasks at runtime. This means the number of tasks does not need to be
```

Comment:
```suggestion
enabling workflows to dynamically create tasks at runtime. This means the number of tasks does not need to be predefined.
```
?

Reply:
thanks I guess some content got cut out earlier. Refactored it now.